src/QuickMarcEditor/QuickMarcCreateWrapper.js

```diff
@@ -141,11 +141,10 @@ const QuickMarcCreateWrapper = ({
           showCallout({ messageId: 'ui-quick-marc.record.saveNew.success' });
 
           if (marcType === MARC_TYPES.BIB && recordHasLinks(formValuesForCreate.fields)) {
-            saveLinksToNewRecord(mutator, externalId, formValuesForCreate)
-              .finally(() => redirectToRecord(externalId, instanceId));
-          } else {
-            redirectToRecord(externalId, instanceId);
+            await saveLinksToNewRecord(mutator, externalId, formValuesForCreate);
```

Comment:
I think if links update request is failed we'll get an uncaught exception. Previously `finally` handled it, but with `await` there needs to be an additional wrapping `try catch` statement

Reply:
It is already in the `try catch` statement, the errors will be caught [here](https://github.com/folio-org/ui-quick-marc/pull/656/files#diff-c020e13bb136b39c646b13f11e5c18e198ab63e2dee7dded2d940c611b70209dR148).